src/dev/ci_setup/setup.sh

```diff
@@ -180,3 +180,23 @@ if [ "$GIT_CHANGES" ]; then
   echo -e "$GIT_CHANGES\n"
   exit 1
 fi
+
+###
+### github-checks-reporter kill switch. Remove to disable
+###
+export CHECKS_REPORTER_ACTIVE=true
```

Comment:
I feel like we should default this to `false`, and only set it to try when `[[ "$JOB_NAME" = "elastic+kibana+pull-request"* ]]`, then we don't have to check the job name in multiple places and can just rely on this environment variable.